README.md

```diff
@@ -18,7 +18,6 @@ the end to end tests to work.
 
 You can then run: "poetry run pytest" to execute all the currently defined tests. 
 
-
```

Comment:
You should update the README with instructions for registering the app in GitHub, what redirect URL to use etc.
This is something each developer would need to do unless there's a shared one. Either way they need to add it to their .env file, but currently they have to read the code to work that out.